README.md

```diff
@@ -12,24 +12,28 @@
   <a href="https://discord.gg/Gcm8BbTaAj">                                <img alt="Rerun Discord"  src="https://img.shields.io/discord/1062300748202921994?label=Rerun%20Discord"> </a>
 </h1>
 
-# Rerun: Visualization infrastructure for computer vision.
+# An SDK for building time aware visualizations of multimodal data
 
-Use one of our logging APIs (Python or Rust) to log rich data, such as images and point clouds, to the Rerun Viewer, where it is visualized live or after the fact.
+Use the Rerun SDK (available in Python, Rust, and soon C++) to log data like images, tensors, point clouds, and text. Logs are streamed to the Rerun Viewer for live visualization or to file for later use.
 
 ```py
-import rerun as rr
+import rerun as rr  # pip install rerun-sdk
 
-rr.init("rerun_example_app", spawn = True) # Spawn a Rerun Viewer and stream log events to it
+rr.init("rerun_example_app")
 
-rr.log_image("rgb_image", image)
-rr.log_points("points", positions)
-rr.log_rect("car", bbox)
+rr.connect()  # Connect to a remote viewer
+# rr.spawn()  # Spawn a child process with a viewer and connect
+# rr.save("recording.rrd")  # Stream all logs to disk
+
+# Associate subsequent data with 42 on the “frame” timeline
+rr.set_time_sequence("frame", 42))
+
+# Log colored 3D points to the entity at `path/to/points`
+rr.log("path/to/points", rr.Points3D(positions, colors=colors)
 …
 ```
+![rerun-github-hero](https://github.com/rerun-io/rerun/assets/2624717/c4900538-fc3a-43b8-841a-8d226e7b5a2e)
```

Comment:
Why no longer centered?

And what is up with that alt-text ("rerun-github-hero") ?

Reply:
I just dragged it in and forgot to update the alt-text. I'll try centering it again